.github/workflows/build_release.yml

```diff
@@ -0,0 +1,97 @@
+name: Build Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [linux-16.04, macos-latest, windows-latest]
```

Comment:
why is this one not `ubuntu-latest`?

Reply:
I figured for max binary compat we should to the release build on an older distro